riscv/encoding.h

```diff
@@ -954,9 +954,9 @@
 #define MATCH_SINVAL_VMA 0x16000073
 #define MASK_SINVAL_VMA  0xfe007fff
 #define MATCH_SFENCE_W_INVAL 0x18000073
-#define MASK_SFENCE_W_INVAL  0xfff07fff
+#define MASK_SFENCE_W_INVAL  0xffffffff
 #define MATCH_SFENCE_INVAL_IR 0x18100073
-#define MASK_SFENCE_INVAL_IR  0xfff07fff
+#define MASK_SFENCE_INVAL_IR  0xffffffff
```

Comment:
This looks suspicious...was it incorrect before?

Reply:
How is this resolved? It's still there in 236fd642967bddeece5312b6a8fa52c5ab693b38

Reply:
Via riscv-opcodes that generates this file.